lib/services/local/users.go

```diff
@@ -62,6 +62,8 @@ var GlobalSessionDataMaxEntries = 5000 // arbitrary
 type IdentityService struct {
 	backend.Backend
 	log logrus.FieldLogger
+	// ServiceClock is used in tests to desync the service clock from the backend clock.
+	ServiceClock clockwork.Clock
```

Comment:
Could we avoid making this public? Maybe use a test-visible-only setter?

Reply:
Thanks for the reminder, I forgot this was possible. Time to go edit some other open PRs...